pkg/orchestration/wiring/wiringutil/knownshapes/zz_generated.deepcopy.go

```diff
@@ -41,6 +66,13 @@ func (in *BindableEnvironmentVariables) DeepCopyShape() wiringplugin.Shape {
 func (in *BindableEnvironmentVariablesData) DeepCopyInto(out *BindableEnvironmentVariablesData) {
 	*out = *in
 	in.BindableShapeStruct.DeepCopyInto(&out.BindableShapeStruct)
+	if in.Vars != nil {
+		in, out := &in.Vars, &out.Vars
+		*out = make(map[string]string, len(*in))
+		for key, val := range *in {
+			(*out)[key] = val
+		}
+	}
```

Comment:
@benbarclay This is why we need post commit-hooks :smile:

Reply:
We need fuzzing tests for shapes. Like kube has for type structs.

Reply:
Also, `BindableEnvironmentVariables` constructor should have a mandatory parameter for vars to make it a compilation error if that field was not initialized

Reply:
IIRC this was generated from `make generate`. Should I remove this section?